src/main/webapp/app/exercises/shared/team/team-participation-table/team-participation-table.component.html

```diff
@@ -34,7 +34,7 @@
                     </span>
                 </ng-template>
                 <ng-template ngx-datatable-cell-template let-value="value" class="me-5">
-                    {{ value ? (moment(value) | artemisDate: 'short-date') : ('artemisApp.exercise.dateNotSet' | artemisTranslate) }}
+                    {{ value ? (dayjs(value) | artemisDate: 'short-date') : ('artemisApp.exercise.dateNotSet' | artemisTranslate) }}
```

Comment:
I suggest not exposing the date API to the template

Reply:
I created a [confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Code+Review) for general implementation/style aspects which can be handled separately after discussion how to proceed with your suggestions.